templates/terraform/sweeper_file.go.erb

```diff
@@ -0,0 +1,115 @@
+<%= lines(autogen_notice :go) -%>
+
+package google
+
+import (
+  "fmt"
+  "testing"
+  "strings"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+<%
+sweeper_name = product_ns + object.name
+wrap_path = object&.nested_query&.keys&.first || object.collection_url_key
```

Comment:
This feels like `collection_key` or `collection_url_key` to me- what does `wrap_path` mean in this context?

Reply:
Borrowed from inspec [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/13bbcc992187a2027276b93af81b9da479811663/templates/inspec/plural_resource.erb#L55). The API responses have the actual list of resources under a key like `items`. If it is some non-standard key, then `collection_url_key` is set to the key. There are probably a couple of resources that need it, I'm still verifying. But that is easily added in another PR

Reply:
Cool, consistency with InSpec trumps my thoughts